lib/ansible/module_utils/service.py

```diff
@@ -147,9 +147,7 @@ def fork_process():
             os._exit(0)
 
         # get new process session and detach
-        sid = os.setsid()
-        if sid == -1:
-            raise Exception("Unable to detach session while daemonizing")
```

Comment:
Can `daemonize` just catch OSError and RuntimeError now when calling this function?